docs/UserGuide.md

```diff
@@ -47,7 +47,9 @@ Example:
  
 ### Adding a task: `add`
 Adds a task into the task manager.<br>
-Format: `add NAME [from/at START_DATE START_TIME][to/by END_DATE END_TIME][repeat every RECURRING_INTERVAL][-PRIORITY]`
+Format: `add NAME, [from/at START_DATE START_TIME][to/by END_DATE END_TIME][repeat every RECURRING_INTERVAL][-PRIORITY]`
+
```

Comment:
Not sure if we should put `,` because if the task only has NAME and -PRIORITY, don't need `,`